packages/app-desktop/gui/NoteEditor/NoteBody/CodeMirror/CodeMirror.tsx

```diff
@@ -182,13 +182,17 @@ function CodeMirror(props: NoteBodyEditorProps, ref: any) {
 								}
 							}
 						},
-						insertText: (value: any) => editorRef.current.insertAtCursor(value),
+						insertText: (value: any) => {
+							editorRef.current.insertAtCursor(value);
+							editorRef.current.focus();
+						},
 						attachFile: async () => {
 							const cursor = editorRef.current.getCursor();
 							const pos = cursorPositionToTextOffset(cursor, props.content);
 
 							const newBody = await commandAttachFileToBody(props.content, null, { position: pos });
 							if (newBody) editorRef.current.updateBody(newBody);
+							editorRef.current.focus();
```

Comment:
I would prefer commands not have side-effects, especially since they are used from plugins (I can imagine some cases where a plugin might change text but without wanting to focus the editor). Instead is it not possible to add the behaviour to the toolbar button click event? i.e. every time any button is clicked, we focus the editor, although I guess there should be an exception for some buttons, like Back/Forward and External Edit. What do you think?

Reply:
That makes sense to me, I was viewing these commands as the toolbar button presses but now that we have a plugin system that is an incorrect assumption. The `wrapSelectionWithStrings` function also focuses the editor (it's used by a few of the commands), so I'll remove that as well to enforce consistency.